processing/format_qaqc_driver.py

```diff
@@ -338,29 +361,32 @@ def run(self):
                                 retry = p.get('retry')
                                 if retry >= self.max_retries:
                                     is_qaqc_successful = False
-                                    _log.info(f"Process {p.get('task').uuid} "
-                                              f'{self.max_retries} '
-                                              'retries reached. '
-                                              'Stop running this process')
+                                    error_task = p.get('task')
+                                    error_msg = (f"Process {error_task.uuid} "
+                                                 f'{self.max_retries} '
+                                                 'retries reached. '
+                                                 'Stop running this process')
+                                    _log.info(error_msg)
+                                    self.blacklist_uuid.append(error_task.uuid)
```

Comment:
What is the blacklist here?

Reply:
blacklist tokens (uuids) are the tokens we won't pick up again in the next round of getting new rows in the upload log table because, in the last round, we can't process those tokens and the error is reported to AMP.